server_spec.rb

```diff
@@ -0,0 +1,234 @@
+require_relative 'server'
```

Comment:
I would recommend moving this file to '/spec/' as it's standard rspec convention. Or you need to have a `.rspec` file which override the convention so that gem running rspec in this project it finds the spec files.